solarwinds_apm/apm_config.py

```diff
@@ -54,8 +76,9 @@ def get_oboe_trigger_trace_mode(cls, trigger_trace_mode: str) -> int:
 
 
 class SolarWindsApmConfig:
-    """SolarWinds APM Configuration Class
-    The precedence: in-code keyword arguments > Environment Variables > config file > default values.
+    """Manage SolarWinds APM configuration.
+
+    Configuration precedence: in-code keyword arguments > Environment Variables > config file > default values.
     Note that oboe doesn't read configurations by itself. The Python agent needs to
     read environment variables and/or config files and pass them into oboe. This is
     done only once during the initialization and the properties cannot be refreshed.
```

Comment:
nit: can probably remove the "oboe doesn't read config..." part since there is no longer a separate blackbox c-extension.